lms/lms/api.py

```diff
@@ -449,8 +449,7 @@ def get_certification_categories():
 	for doc in docs:
 		category = doc.course_title if doc.course_title else doc.batch_title
 		if category not in categories:
```

Comment:
this won't work with key value pairs of label and value, you will need to change this condition to check for duplicates